NEWS.md

```diff
@@ -20,7 +21,7 @@ OpenSSL Releases
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
 
-OpenSSL 3.5
+OpenSSL 3.6
```

Comment:
These changes to NEWS.MD are some post-release missing changes, no ? 
Deserve a separated short PR ?